tests/test_agent_registry.py

```
@@ -0,0 +1,459 @@
+            user_id="user-1", framework="openclaw",
+        )
+        payload = verify_registry_token(token, pub)
+        assert payload["sub"] == "agent-20260609-120000"
+        assert payload["iss"] == "taos-registry"
+        assert "iat" in payload
+        assert payload["user_id"] == "user-1"
+        assert payload["framework"] == "openclaw"
+
+    def test_verify_wrong_key_fails(self, tmp_path):
+        import tempfile
+        priv1, pub1 = self._make_keypair(tmp_path)
```

Ruff (0.15.15)

[warning] Unpacked variable `pub1` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)